po/af.po

```diff
@@ -1601,6 +1601,11 @@ msgstr "Skep partisie %1$s (%2$s)"
 msgid "Probing partitions failed"
 msgstr ""
 
+#. TRANSLATORS: error message
+#, fuzzy, c-format
+msgid "Probing partitions on %s failed"
+msgstr "Skep partisie %1$s (%2$s)"
```

Comment:
this looks wrong and broken

Reply:
The string is marked as fuzzy and AFAIK not included in the mo file.